components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -394,6 +416,7 @@ def dump_to_primitive_dict(self):
         d = self.dict()
         d["archive_output"] = self.archive_output.dump_to_primitive_dict()
         # Turn paths into primitive strings
+        d["os_release_file_path"] = str(self._os_release_file_path)
```

Comment:
We use this method to dump the config and then load it in the container, but since this member is private, adding it in the dump is not really necessary. What do you think?

Reply:
Right, I remember adding this type-cast / conversion because Pydantic doesn't know how to dump this field. Now that the field has been changed into a private one, it no longer gets dumped and the conversion is no longer needed.